### ISSUE TITLE
Subsites should handle `uuid` redirection to respective resource viewer page

### ISSUE THREAD

**Issue by dsuren1** [geosolutions-it/geonode-subsites#11]
### Description
Subsites currently doesn't handle urls formed with resource's uuid ex: `/uuid/${id}`

### Expected
urls redirection with resource uuid should be redirected to respective resource viewer page. This is applicable also for `subsites`

**Comment by giohappy** [comment]
good point @dsuren1 but this is tricky. How do we decide which subsite should show the resource?
I don't have an answer at the moment, we need to find a solution.

**Comment by giohappy** [comment]
Forget it, the solution is only a matter of implementing the url inside the subsites module.
The finale result is:

. the normal uuid url (https://mygeonode.com/catalogue/uuid/51023648-6315-4fed-bf09-3415fc776caf) will go to the default catalogue:
   - if it's condifured to be reached by end users it will work as usual
   - otherwise the user will be redirected to the home page
- the uuid url for the subsite will redirect to the subsite viewer
